security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,153 +12,28 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
-[id="cert-manager-operator-release-notes-1.12.1"]
-== Release notes for {cert-manager-operator} 1.12.1
+[id="cert-manager-operator-release-notes-1.13"]
+== Release notes for {cert-manager-operator} 1.13
 
-Issued: 2023-11-15
+Issued: 2024-xx-xx
 
-The following advisory is available for the {cert-manager-operator} 1.12.1:
+The following advisory is available for the {cert-manager-operator} 1.13:
```

Comment:
Operator version is `1.13.0`. Please consider changing it for all occurrences in the text.